boards/st/nucleo_h563zi/board.cmake

```diff
@@ -6,6 +6,11 @@ board_runner_args(pyocd "--target=stm32h563zitx")
 
 board_runner_args(jlink "--device=STM32H563ZI" "--reset-after-load")
 
+board_runner_args(openocd "--tcl-port=6666")
+board_runner_args(openocd --cmd-pre-init "gdb_report_data_abort enable")
+board_runner_args(openocd "--no-halt")
+
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
```

Comment:
Keep `stm32cubeprogrammer` as default in order not to break existing board users.

Reply:
done